### PR TITLE
Devel condor

### PR DESCRIPTION
**Pull request by cjdcordeiro**
added the new RPM and YUM repo for fetching files from github instead of DFS.

P.S.: website has changed to http://cern-cloudinit-modules.web.cern.ch/cern-cloudinit-modules/

### CODE REVIEW

File: etc/cern-cloudinit-modules.repo [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [cerncloudinitrepo]
 name=CERN-CloudInit-Modules
-baseurl=http://chdoming.web.cern.ch/chdoming/package/
+baseurl=https://github.com/cjdcordeiro/cloud-init-cern/tree/devel-condor/rpm
```
Comment (cinquo) [inline etc/cern-cloudinit-modules.repo:3]: Please, use the main repo on http://cern-cloudinit-modules.web.cern.ch/

File: etc/cern-cloudinit-modules.spec [hunk 2]
```diff
@@ -15,8 +15,9 @@ This RPM copies the cloud config modules of cvmfs, Ganglia and Condor to its res
 
 %pre
 echo "Downloading the modules..."
-wget chdoming.web.cern.ch/chdoming/cern-cloudinit-modules.tar
-tar xvf cern-cloudinit-modules.tar
+wget https://raw.github.com/cjdcordeiro/cloud-init-cern/devel-condor/cloudinit/config/cc_condor.py
```
Comment (cinquo) [inline etc/cern-cloudinit-modules.spec:18]: Use my main repo. Use git client and get all modules instead of single file download.

**Comment by cinquo** [comment]
In  addition to these comments there is the fact that the changes are in condor-devel, while they should be under devel branch.

**Comment by cinquo** [comment]
Plus, I can't make it working with your repo file:

<pre>
Loaded plugins: changelog, kernel-module, priorities, protectbase, refresh-packagekit, security, tsflags, versionlock
ai6                                                                                                                                                                                                         | 2.5 kB     00:00     
ai6-iaas-ibex-os                                                                                                                                                                                            | 2.5 kB     00:00     
cerncloudinitrepo                                                                                                                                                                                           |  36 kB     00:00     
https://github.com/cjdcordeiro/cloud-init-cern/tree/devel-condor/rpm/repodata/repomd.xml: [Errno -1] Error importing repomd.xml for cerncloudinitrepo: Damaged repomd.xml file
Trying other mirror.
Error: Cannot retrieve repository metadata (repomd.xml) for repository: cerncloudinitrepo. Please verify its path and try again
</pre>

File: etc/cern-cloudinit-modules.repo (hunk 1 above)
Comment (cjdcordeiro) [inline etc/cern-cloudinit-modules.repo:3]: That was fixed on this last pull.

File: etc/cern-cloudinit-modules.spec (hunk 2 above)
Comment (cjdcordeiro) [inline etc/cern-cloudinit-modules.spec:18]: Already fixed on this last pull. It's now 'wget' the three modules, one by one, from your repo, devel branch

**Comment by cjdcordeiro** [comment]
I think I the issues were reviewed and fixed on the last pull

**Pull request closed without merging**